src/main/java/uk/gov/pay/connector/tasks/HistoricalEventEmitterWorker.java

```diff
@@ -118,11 +138,31 @@ private void offerPaymentStateTransitionEvents(long currentId, ChargeEventEntity
         } else {
             logger.info("[{}/{}] - found - emitting {} for charge event [{}] ", currentId, maxId, event, chargeEventEntity.getId());
             stateTransitionQueue.offer(transition);
-            persistEvent(event);
+            persistEventEmitRecord(event);
+        }
+    }
+
+    private void processManualPaymentEvents(List<ChargeEventEntity> chargeEventEntities) {
```

Comment:
I think, we can be specific here to be clear for the reader
```suggestion
    private void processPaymentDetailsEnteredEvents(List<ChargeEventEntity> chargeEventEntities) {
```